ci/scripts/r_docker_configure.sh

```diff
@@ -74,20 +74,105 @@ if [[ -n "$DEVTOOLSET_VERSION" ]]; then
   $PACKAGE_MANAGER install -y "devtoolset-$DEVTOOLSET_VERSION"
 fi
 
+case "$PACKAGE_MANAGER" in
```

Comment:
Why does the R setup need Python installed now?

Reply:
Yeah I think I'd like that inside a ARROW_GCS check, just to be clear that that's why we would need Python.

Reply:
OK. I'll do it.